scripts/spm-integration-test.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
+
+set -e -uxf -o pipefail
+wait_seconds=5
+retry_count=10
```

Comment:
These are not needed at the whole file level, as they are only used in one function. And also let's change them to the timeout approach, not iterations.